source/cellular/known-working-3g-4g-modems.rst

```diff
@@ -130,6 +127,11 @@ Modems reported to work as Ethernet devices
   system does not have an overlapping network.
 - ZTE MF915 LTE modem (T-Mobile)
 - ZTE MF975S
+- Huawei E3372h - Command to switch to the correct mode:
```

Comment:
You can use `::` at the end of this line and then you do not need the `::` on the next line. Ending a line with `::` will start the code block.